torchvision/datasets/folder.py

```diff
@@ -75,7 +75,7 @@ class DatasetFolder(VisionDataset):
             E.g, ``transforms.RandomCrop`` for images.
         target_transform (callable, optional): A function/transform that takes
             in the target and transforms it.
-        is_valid_file (callable, optional): A function that takes path of an Image file
+        is_valid_file (callable, optional): A function that takes path of an file
```

Comment:
can you replace this by
```
A function that takes path of a file
```
given that now you need to use `a file`?

Reply:
done and remove some unwanted _